microlith/loki/alerting/couchbase/couchbase-rules.yaml

```diff
@@ -273,3 +273,19 @@ groups:
           title: "Large Number of Dropped Ticks Detected (node: {{ $labels.hostname }})"
           description: Couchbase Server nodes regularly send heartbeat ticks to each other. Dropped ticks are usually a sign of resource contention.
           remediation: Increase number of CPUs available to Couchbase Server or, if you are running a virtualised environment, check for VM overcommitment.
+
+      - alert: CB90067-serviceRollback
+        expr: |
+          count_over_time( {file="memcached.log"} |= "requires rollback to seqno:0" != "eq_dcpq:replication" [24h]) > 0
+        labels:
+          job: couchbase_fluent_bit
+          kind: node
+          severity: alert
+          health_check_id: CB90067
+          health_check_name: serviceRollback
+          cluster: '{{ $labels.cluster }}'
```

Comment:
`cluster` may not always exist as a label - have you tested the rule to make sure that this is correct?

Reply:
Ah whoops I must have forgotten to remove that, will do now

Reply:
We do want the cluster label there (otherwise it won't even show up on dashboards), but I think the actual Loki label may be `couchbase.cluster` rather than `cluster` - look at how the other rules handle this though because I don't remember off the top of my head.